brewchat/src/main/java/com/example/brewchat/Application.java

```diff
@@ -42,6 +42,7 @@ public void onCreate() {
                 getString(R.string.app_id),
                 getString(R.string.auth_key),
                 getString(R.string.auth_secret));
+        chatService.createSession();
```

Comment:
Happens when you lose the session? Where should It be created again if you loose the Internet connection?